src/cli.rs

```diff
@@ -63,6 +63,9 @@ pub enum Commands {
     /// Edge App related commands.
     #[command(subcommand)]
     EdgeApp(EdgeAppCommands),
+    // For generating `docs/CommandLineHelp.md`.
```

Comment:
well it should still be ///.. Though I guess it's hidden anyhow... So might not matter